### ISSUE TITLE
Context menu of webapps only in english

### ISSUE THREAD

**Issue by jakoblm** [linuxmint/mint20.1-beta#29]
Dear Mint Team,

Linux Mint 20.1 Cinnamon: The context menu (right mouse button) of a created webapp is only displayed in English and is not translated into German. 
**EDIT:** I mean the context menu within an open webapp (not in the respective entry of the menu).

Are translations into other languages already available?

**Comment by MenthaSuaveolens** [comment]
I can't confirm. It works ok in dutch.
![WebApp](https://user-images.githubusercontent.com/47174957/102795103-a664ec80-43ac-11eb-85d2-a25d383df458.png)

**Comment by mtwebster** [comment]
This would affect every item in the menu, not just webapps.

Which menu applet is this? Is it the one that came with cinnamon or an applet installed after? (Look in the right-click->about popup if you right-click on the panel button - the default is menu@cinnamon-org).

**Comment by jakoblm** [comment]
Apparently I have expressed myself in a misleading way, sorry! I meant the context menu **within an open webapp** (not in the respective entry of the menu). 
I have set the system language to German. However, I test this from the live system without installing updates or rebooting. Maybe that's the reason? 
![Bildschirmfoto vom 2020-12-21 23-25-52](https://user-images.githubusercontent.com/51761218/102828036-ff9c4280-43e3-11eb-9eb8-2ae9f6098457.png)
